reference-architecture/azure-ansible/bastion.sh

```diff
@@ -0,0 +1,337 @@
+yum -y remove RHEL7
+rm -f /etc/yum.repos.d/rh-cloud.repo
+# Found that wildcard disable not working all the time - make sure
+yum-config-manager --disable epel
+yum-config-manager --disable epel-testing
+subscription-manager register --username $RHNUSERNAME --password $RHNPASSWORD
+subscription-manager attach --pool=$RHNPOOLID
+subscription-manager repos --disable="*"
+subscription-manager repos     --enable="rhel-7-server-rpms"     --enable="rhel-7-server-extras-rpms"
+subscription-manager repos     --enable="rhel-7-server-ose-3.3-rpms"
+yum -y install atomic-openshift-utils
+yum -y install git net-tools bind-utils iptables-services bridge-utils bash-completion httpd-tools
+yum -y install docker
+touch /root/.updateok
+sed -i -e "s#^OPTIONS='--selinux-enabled'#OPTIONS='--selinux-enabled --insecure-registry 172.30.0.0/16'#" /etc/sysconfig/docker
+                                                                                         
+cat <<EOF > /etc/sysconfig/docker-storage-setup
+DEVS=/dev/sdc
+VG=docker-vg
+EOF
+
+docker-storage-setup                                                                                                                                    
+systemctl enable docker
+systemctl start docker
+
+
+cat <<EOF > /etc/ansible/hosts
+[OSEv3:children]
+masters
+etcd
+nodes
+misc
+
+[OSEv3:vars]
+azure_resource_group=${RESOURCEGROUP}
+rhn_pool_id=${RHNPOOLID}
+openshift_install_examples=true
+deployment_type=openshift-enterprise
+openshift_master_identity_providers=[{'name': 'htpasswd_auth', 'login': 'true', 'challenge': 'true', 'kind': 'HTPasswdPasswordIdentityProvider', 'filename': '/etc/origin/master/htpasswd'}]
+
+# default selectors for router and registry services
+openshift_router_selector='region=infra'
+openshift_registry_selector='region=infra'
+
+ansible_become=yes
+ansible_ssh_user=${AUSERNAME}
+remote_user=${AUSERNAME}
+
+openshift_master_default_subdomain=${ROUTEREXTIP}.xip.io 
+openshift_use_dnsmasq=False
+openshift_public_hostname=${RESOURCEGROUP}.trafficmanager.net
+
+openshift_master_cluster_method=native
+openshift_master_cluster_hostname=${RESOURCEGROUP}.trafficmanager.net
+openshift_master_cluster_public_hostname=${RESOURCEGROUP}.trafficmanager.net
+
+# default storage plugin dependencies to install, by default the ceph and
+# glusterfs plugin dependencies will be installed, if available.
+osn_storage_plugin_deps=['iscsi']
+
+[masters]
+master1 openshift_node_labels="{'role': 'master'}"
+master2 openshift_node_labels="{'role': 'master'}"
+master3 openshift_node_labels="{'role': 'master'}"
+
+[etcd]
+master1
+master2
+master3
+
+[nodes]
+master1 openshift_node_labels="{'region':'master','zone':'default'}" 
+master2 openshift_node_labels="{'region':'master','zone':'default'}" 
+master3 openshift_node_labels="{'region':'master','zone':'default'}" 
+node[01:${NODECOUNT}] openshift_node_labels="{'region': 'primary', 'zone': 'default'}"
+infranode1 openshift_node_labels="{'region': 'infra', 'zone': 'default'}"
```

Comment:
can change the nodes to "app" if you think thats sensible. The above is based on documentation, so which is better, stick to doc, or copy other providers?